### PR TITLE
fix: add cpf/cnpj field if it not exists

### PR DESCRIPTION
**Pull request by murilohns**
refactor: unified css file

feature: add mask to cpf/cnpj field

refactor: new function to get customer document_number

### CODE REVIEW

File: API/1.5.x/upload/catalog/view/theme/default/template/payment/pagar_me_cartao.tpl [hunk 1]
```diff
@@ -58,10 +58,12 @@
             <input type="text" id="card_holder_name" value=""/>
         </div>
 
+        <?php if(empty($customer_document_number)) { ?>
         <div class="input-block">
-            <label for="cpf_customer">CPF/CNPJ (somente números)</label>
-            <input type="text" id="cpf_customer" value=""/>
+            <label for="customer_document_number">CPF/CNPJ (somente números)</label>
+            <input type="text" id="customer_document_number" value=""/>
```
Comment (jonatasmaxi) [inline API/1.5.x/upload/catalog/view/theme/default/template/payment/pagar_me_cartao.tpl]: I think required=true here would fit better.

File: API/1.5.x/upload/catalog/controller/payment/pagar_me_boleto.php [hunk 2]
```diff
@@ -62,23 +56,16 @@ public function payment()
         $order_info = $this->model_checkout_order->getOrder($this->session->data['order_id']);
         $customer = $this->model_account_customer->getCustomer($order_info['customer_id']);
 
-        $possibleDocumentFields = array(
-            'cpf', 'cnpj', 'document_number', 'payment_tax_id'
-        );
+        $document_number = $this->getCustomerDocumentNumber() ? $this->getCustomerDocumentNumber() : $this->request->post['document_number'];
```
Comment (leonampd) [inline API/1.5.x/upload/catalog/controller/payment/pagar_me_boleto.php]: You're calling `getCustomerDocumentNumber` twice. I suggest to assign the returned value to a variable and, if it's empty or null, assign the default value

File: API/1.5.x/upload/catalog/controller/payment/pagar_me_cartao.php [hunk 3]
```diff
@@ -94,26 +86,29 @@ public function payment()
 
         $customer = $this->model_account_customer->getCustomer($order_info['customer_id']);
 
-        if ($this->config->get('dados_status')) {
-            if ($customer['cpf'] != '') {
-                $customer_name = $order_info['payment_firstname'] . " " . $order_info['payment_lastname'];
-            } else {
-                $customer_name = $customer['razao_social'];
+        $document_number = $this->getCustomerDocumentNumber() ? $this->getCustomerDocumentNumber() : $this->request->post['document_number'];
```
Comment (leonampd) [inline API/1.5.x/upload/catalog/controller/payment/pagar_me_cartao.php]: The same as above

File: API/1.5.x/upload/catalog/controller/payment/pagar_me_cartao.php [hunk 4]
```diff
@@ -94,26 +86,29 @@ public function payment()
...
-            }
-            $numero = $order_info['payment_numero'];
-            $complemento = $order_info['payment_company'];
-        } else {
-            $customer_name = $order_info['payment_firstname'] . " " . $order_info['payment_lastname'];
-            $numero = 'Sem número';
-            $complemento = '';
+        $documentNumberLenght = strlen($document_number);
+
+        $isCpf = $documentNumberLenght == 11;
```
[the remark this answers is not in the document]
Comment (leonampd) [reply API/1.5.x/upload/catalog/controller/payment/pagar_me_cartao.php]: I got your point but, put this check in a if condition will make the code difficult to read

File: API/1.5.x/upload/catalog/view/theme/default/template/payment/pagar_me_boleto.tpl [hunk 5]
```diff
@@ -34,13 +65,22 @@ $('#button-confirm').bind('click', function (e) {
         });
     });
 
+documentNumberField.keydown(function() {
+
+   if(documentNumberField.val().length > 14) {
+     documentNumberField.unmask();
+     documentNumberField.mask("99.999.999/9999-99");
+   } else{
+     documentNumberField.mask("999.999.999-990000");
```
[the remark this answers is not in the document]
Comment (murilohns) [reply API/1.5.x/upload/catalog/view/theme/default/template/payment/pagar_me_boleto.tpl]: It must have the same CNPJ length

File: API/1.5.x/upload/catalog/view/theme/default/template/payment/pagar_me_cartao.tpl (hunk 1 above)
Comment (murilohns) [reply API/1.5.x/upload/catalog/view/theme/default/template/payment/pagar_me_cartao.tpl]: We don't need it, if the document number isn't filled, the error "Número de documento está faltando" will be thrown.

**Comment by murilohns** [comment]
@leonampd and @jonatasmaxi, can u review again?

**Pull request merged**